### PR TITLE
#163518636 Setup Redux 

### PR DESCRIPTION
**Pull request by YvesIraguha**
#### What does this PR do?
- Setup redux 
#### Description of Task to be completed?
- Include redux in the application
- Connect react with redux 
- Include redux-thunk in the application
- Create dummy actionCreator, store, and reducer 
- Ensure that dummy actionCreator, store, and reducer is working
#### How should this be manually tested?
- Fork the repo and check out the branch `ch-setup-redux-163518636`
- In the root folder run `yarn` to install all dependencies
- Run `yarn start` to start the project
- The browser should be opened. Click the button label `Fetch more articles`. 
- You should see more articles being added to the page. 

#### What are the relevant pivotal tracker stories?
#163518636

### CODE REVIEW

**Review by espoirMur — APPROVED** [review]
LGTM

**Review by abayo-luc — APPROVED** [review]

File: src/views/Home.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+};
+
+const connectedArticles = ({ articles }) => (
+  <ul>
+    {articles.map((el, index) => (
+      <div
+        key={index}
+        style={{ borderBottom: '1px solid red', width: '700px' }}
+      >
+        <h4>{el.title}</h4>
```
Comment (puritybirir) [inline src/views/Home.js]: Better Variable Naming. What does el represent?

File: src/views/Article.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Fetch some articles from dummy app</h1>
+
+        <button
+          onClick={this.handleFetch}
+          style={{
```
Comment (puritybirir) [inline src/views/Article.js]: Why do we have internal styling

File: src/views/Home.js [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+const mapStateToProps = state => {
+  return { articles: state.articles };
+};
+
+const connectedArticles = ({ articles }) => (
+  <ul>
+    {articles.map((el, index) => (
+      <div
+        key={index}
+        style={{ borderBottom: '1px solid red', width: '700px' }}
```
Comment (puritybirir) [inline src/views/Home.js]: Internal styles

File: src/views/Home.js [hunk 4]
```diff
@@ -1,11 +1,23 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
```
Comment (puritybirir) [inline src/views/Home.js]: Component is imported but not used

**Review by puritybirir — CHANGES_REQUESTED** [review]
Additional changes:

- Change
- Ensure all styles are in our CSS files and not internal.

 the views folder to  components

**Review by puritybirir — APPROVED** [review]

**Review by puritybirir — APPROVED** [review]

**Pull request merged**